### PR TITLE
Gitlab integration

### PR DESCRIPTION
**Pull request by timgent**

### CODE REVIEW

File: kubernetes_deployment/deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ spec:
         env:
         - name: GITHUB_REPOS
           value: {{.GITHUB_REPOS}}
+        - name: DRONE_GH_SERVER
+          value: https://drone.digital.homeoffice.gov.uk
```
Comment (chrisns) [inline kubernetes_deployment/deployment.yaml]: can you not specify also `DRONE_GL_SERVER` & `GITLAB_SERVER`?

File: lib/drone.rb [hunk 2]
```diff
@@ -1,44 +1,36 @@
...
+    } }
+  end
 
-def get_latest_build_for_each_branch(json)
-  json.group_by { |b| b['branch'] }.map { |branch| branch[1].max_by { |build| build['started_at'] } }
-end
+  def Drone.translate_status_to_class(status)
+    statuses = {
+        'success' => 'passed',
+        'running' => 'pending',
```
Comment (chrisns) [inline lib/drone.rb]: if you're defaulting to `pending` then do you need to specify that `running` will also map to `pending`?

Comment (chrisns) [inline lib/drone.rb]: consider constants rather than strings for statuses?

Comment (timgent) [inline lib/drone.rb]: All updated as discussed

File: kubernetes_deployment/deployment.yaml (hunk 1 above)
Comment (timgent) [inline kubernetes_deployment/deployment.yaml]: Done

**Comment by timgent** [comment]
Issue raised as discussed - https://github.com/UKHomeOffice/ho-example-dashboard/issues/9

File: spec/test_config.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+TEST_CONFIG = {
+    :github_login => 'gh_username',
```
Comment (chrisns) [inline spec/test_config.rb]: redundant?

**Pull request merged**